messages/errorMessages.ts

```diff
@@ -5,3 +5,11 @@ export const TELLING_MALE_TO_GIVE_BIRTH = "Only females can give birth";
 export const GIVING_BIRTH_WITHOUT_HOME =
   "Can't give birth without having a home";
 export const EMPTY_COLOR_ERROR_MESSAGE = "Color cannot be empty";
+export const NEGATIVE_NUTS_ADDED = "Cannot add a negative number of nuts";
+export const NEGATIVE_HOLE_SIZE = "Hole size must be greater than zero";
+export const NEGATIVE_TREE_AGE = "Tree age must be greater than zero";
+export const NEGATIVE_VALUES = "Both values must be positive!";
```

Comment:
some sentences finish in exclamation sign , others just finish. Maybe we need to be a bit more consistent